src/planscape/bin/load_restrictions.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# this script usually will be executed by make load-restrictions and it will cd into src/planscape
+python manage.py load_restrictions --input-file ../../data/wilderness_area.shp --type WILDERNESS_AREA
+python manage.py load_restrictions --input-file ../../data/private_lands.shp --type PRIVATE_LANDS
+python manage.py load_restrictions --input-file ../../data/state_parks.shp --type STATE_PARKS
+python manage.py load_restrictions --input-file ../../data/national_parks.shp --type NATIONAL_PARKS
+# python manage.py load_restrictions --input-file data/tribal_lands.shp --type TRIBAL_LANDS
+# python manage.py load_restrictions --input-file data/national_forests.shp --type NATIONAL_FORESTS
```

Comment:
One day I suppose we'd want to make this less hard-coded, but let's just get this out for now.